### PR TITLE
Feature/additional pages

### LINKED ISSUE #7: Create pages for Moodle,  Contact(?), User info

### PR DESCRIPTION
**Pull request by aaronsun1030**
Created the moodle, student info, and course info pages
Fixes #7

### CODE REVIEW

File: src/ui/course_info.py
Comment (karthikRavichandran) [inline src/ui/course_info.py:1]: image

Can you make this part of UI can look better? Try to add tree-like structure

Comment (karthikRavichandran) [reply src/ui/course_info.py:1]: @aaronsun1030 TODO

Comment (aaronsun1030) [reply src/ui/course_info.py:1]: I converted the display to markdown so it looks better now: 
image
I think we can fix details like this if we have extra time, but we have to focus on meeting deliverables like testing most importantly.

**Review by karthikRavichandran — APPROVED** [review]
Thanks. Now it looks good. I added Drop down in Course info section

**Pull request merged**